tests_new/integration_tests/client.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import boto3
+import os
+from munch import DefaultMunch
+from dataall.base.aws.parameter_store import ParameterStoreManager
+
+
+ENVNAME = os.getenv('ENVNAME', 'dev')
+
+
+class Client:
+    def __init__(self, username, password):
+        self.username = username
+        self.password = password
+        self.token = self._get_jwt_token()
+
+    def query(self, query: str):
+        endpoint = (
+            os.getenv('API_ENDPOINT', False)
+            if os.getenv('API_ENDPOINT', False)
+            else ParameterStoreManager.get_parameter_value(
+                region=os.getenv('AWS_REGION', 'eu-west-1'), parameter_path=f'/dataall/{ENVNAME}/apiGateway/backendUrl'
+            )
+        )
```

Comment:
I think the `API_ENDPOINT` variable is enough here. If devs want to pull the endpoint from ParameterStore they can use AWS CLI and a .envrc file to automate it.